server/api/utils.js

```diff
@@ -34,6 +35,28 @@ router.post("utils.gc", async (ctx) => {
 
   log(`Destroyed ${countDeletedDocument} documents`);
 
+  log(`Expiring all the collection export older than 30 days…`);
+
+  const exports = await FileOperation.findAll({
+    where: {
+      type: "export",
+      createdAt: {
+        [Op.lt]: subDays(new Date(), 30),
+      },
+      state: {
+        [Op.ne]: "expired",
+      },
+    },
+  });
+
+  await Promise.all(
+    exports.map(async (e) => {
+      e.state = "expired";
```

Comment:
It might be nice to make this a model method, so we can just do `model.expire()`